### ISSUE TITLE
Drink Image

### ISSUE THREAD

**Issue by jeroenmaes** [JonckheereM/Public#6]
Drink Image gaat toch wel héél vaak de mist in...

VB: http://publicapp.tk/drinks/104
Nen zak frieten???
- Minder gekke bieren toelaten?
- ...

**Comment by mmphs** [comment]
Inderdaad, misschien wat van die exotische bieren eruit smijten of alleszins niet gebruiken op de verdediging
